packages/app/src/scenes/widgets/stores/MusicPlayerStore/MusicPlayerStore.ts

```diff
@@ -9,7 +9,9 @@ const MusicPlayerStore = types.compose(
     .model('MusicPlayerStore', {
       musicPlayerWidget: types.optional(Dialog, {}),
       playlist: types.optional(Playlist, {}),
-      musicPlayer: types.optional(MusicPlayer, {})
+      musicPlayer: types.optional(MusicPlayer, {}),
+      isPausedInSpace: types.optional(types.boolean, false),
```

Comment:
Why overcomplicate yourself?
```suggestion
      isPausedInSpace: false,
```